beater/handlers.go

```diff
@@ -103,6 +103,14 @@ var (
 		code:    http.StatusAccepted,
 		counter: counter("response.valid.accepted"),
 	}
+	internalErrorCounter  = counter("response.errors.internal")
```

Comment:
In v2 we introduced a ['response.errors.other'](https://github.com/elastic/apm-server/blob/v2/beater/common_handlers.go#L74). It should be the same as `internalErrorCounter`, @graphaelli @roncohen whoever merges first, please ensure to align those two counters.